src/AppBundle/Controller/BlogController.php

```diff
@@ -34,16 +34,24 @@
 class BlogController extends Controller
 {
     /**
-     * @Route("/", defaults={"page": "1"}, name="blog_index")
-     * @Route("/page/{page}", requirements={"page": "[1-9]\d*"}, name="blog_index_paginated")
+     * @Route("/", defaults={"page": "1", "_format"="html"}, name="blog_index")
+     * @Route("/rss.xml", name="blog_rss", defaults={"page": "1", "_format"="xml"})
```

Comment:
Could we keep the order of arguments, i.e. move `name="blog_rss"` to the end like in others? 😇